README.md

```diff
@@ -38,8 +33,7 @@ git clone https://github.com/awslabs/fhir-works-on-aws-deployment.git
  
 ## Architecture
 
-The system architecture consists of multiple layers of AWS serverless services. The endpoint is hosted using API Gateway. The database and storage layer consists of Amazon DynamoDB and S3, with Elasticsearch as the search index for the data written to DynamoDB. The endpoint is secured by API keys and Cognito for user-level authentication and user-group authorization. The diagram below shows the FHIR server’s system architecture components and how they are related.
-![Architecture](resources/architecture.png)
+The system architecture consists of multiple layers of AWS serverless services. The endpoint is hosted using API Gateway. The storage layer is backed by an Integration Transform in which FHIR Works makes REST API requests to. The FHIR endpoint is secured by API keys and Cognito for user-level authentication and user-group authorization. 
```

Comment:
> The FHIR endpoint is secured by API keys and Cognito for user-level authentication and user-group authorization. 

Is this part true?

Reply:
Yep, clients to FWoA API still access the API in the same way as before. Under the hood FWoA uses IAM auth to make requests to IntTran